package.json

```diff
@@ -45,6 +45,7 @@
     "level-sublevel": "~6.6.5",
     "mongodb": "^6.11.0",
     "node-forge": "^1.3.1",
+    "path-scurry": "^2.0.0",
```

Comment:
what's this ?

Reply:
some dependency needed to build `node-fcntl`, otherwise it would fail:
```
gyp ERR! configure error 
gyp ERR! stack Error: Cannot find module 'path-scurry'
gyp ERR! stack Require stack:
gyp ERR! stack - /home/runner/work/Arsenal/Arsenal/node_modules/cacache/node_modules/glob/dist/commonjs/glob.js
gyp ERR! stack - /home/runner/work/Arsenal/Arsenal/node_modules/cacache/node_modules/glob/dist/commonjs/index.js
gyp ERR! stack - /home/runner/work/Arsenal/Arsenal/node_modules/cacache/lib/util/glob.js
```

Reply:
oh maybe it should be in node-fcntl then

Reply:
let me make a PR over there then.

Reply:
https://github.com/scality/node-fcntl/pull/16

Reply:
some problem happened with the lockfile, it's old can you rather

```
yarn remove fcntl
yarn add github:scality/node-fcntl#0.3.0
```

it should fix the issue without the need to add the dependency yourself

Reply:
indeed, thanks!